assets/styles/base/_base.scss

```diff
@@ -44,5 +44,5 @@ a {
 }
 
 .connected-base {
-  padding: 31px 100px 50px 50px;
+  padding: 31px 100px 50px;
 }
```

Comment:
All code I saw in the demo app is in the form :

    padding-top: xxx
    padding-left: xxx
    […]

I think that is a good thing for the readability. What do you think about that ?

Reply:
Mixed feelings:
On the one hand yes it's not easily obvious with the shorthand version what are the parameter order
On the other, it is something simple, and our CSS linter prefers the shorthand version.
So I don't know and want other's opinions on it as well

Reply:
The team consensus is to keep shorthand versions